Core/HLE/scePsmf.cpp

```diff
@@ -1229,6 +1229,10 @@ static int _PsmfPlayerSetPsmfOffset(u32 psmfPlayer, const char *filename, int of
 	psmfplayer->videoWidth = buf[142] * 16;
 	psmfplayer->videoHeight = buf[143] * 16;
 
+	// Reflect these values to memory - LocoRoco2 reads the height directly here.
+	Memory::Write_U32(psmfplayer->videoWidth, psmfPlayer + 0xBC);
```

Comment:
I think we may need to allocate a struct, and write that pointer to `psmfplayer` on create.  If this works, I bet it's dereferencing that pointer and using the private structure, which we hadn't been emulating yet.

AFAIK, you can do this:

```c
PsmfPlayerData **p1 = nullptr;
scePsmfPlayerCreate(p1, createParams);

PsmfPlayerData **p2 = p1;
scePsmfPlayerSetPsmf(p2, "video.pmf");

scePsmfPlayerStart(p1, startParams, 0);
```

And really, you can only create one psmf player at a time...

Won't be able to double check until some hours from now.

-[Unknown]